### PR TITLE
Clean up some obsolete ARMv7 bits and add ARMv7 UEFI support

### PR DESCRIPTION
**Pull request by nullr0ute**
This is linked/dependent on https://github.com/storaged-project/blivet/pull/715

This patch does some cleanups of some old unused ARMv7 code paths, and adds support for UEFI on ARMv7

### CODE REVIEW

File: pyanaconda/bootloader.py [hunk 1]
```diff
@@ -1917,6 +1917,14 @@ def __init__(self):
         super().__init__()
         self._packages64 = ["grub2-efi-aa64", "shim-aa64"]
 
+class ArmEFIGRUB(EFIGRUB):
+    _serial_consoles = ["ttyAMA", "ttyS"]
```
Comment (ausil) [inline pyanaconda/bootloader.py]: should we attempt to get all the possible serial consoles in here?

Comment (nullr0ute) [reply pyanaconda/bootloader.py]: No, at least no in this patch series. I plan on having some more cleanup for ARM bits but as a separate PR, but the two above cover a lot of devices.

**Comment by nullr0ute** [comment]
@M4rtinK any discussions

**Comment by nullr0ute** [comment]
Now that  storaged-project/blivet#715 is merged @M4rtinK can we get this reviewed and a build complete for F-29+ please?

**Comment by poncovka** [comment]
Jenkins, test this please.

**Comment by poncovka** [comment]
Please, remove the unused import:

```
************* Module ../pyanaconda/payload/__init__.py
W0611(unused-import): 56,0: : Unused platform imported from pyanaconda.platform
```

You might also want to rebase the pull request to `f29-devel`, if you want the patch in Fedora 29.

**Comment by nullr0ute** [comment]
f29-devel didn't exist when I did this PR :)

**Comment by nullr0ute** [comment]
Right, that should be fixed, and rebased on f29-devel (presumably that will also land into F-30 too)

**Comment by nullr0ute** [comment]
@poncovka can you do another Jenkins run please?

**Comment by poncovka** [comment]
I have just realized that we have not set up tests for f29-devel yet :-) The tests pass when I run them manually.

**Pull request merged**